plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java

```diff
@@ -1075,8 +1075,9 @@ public final class CoreMetrics {
    * @deprecated since 10.6
    */
   @Deprecated(since = "10.6")
-  public static final Metric<Integer> HIGH_IMPACT_ACCEPTED_ISSUES = new Metric.Builder(HIGH_IMPACT_ACCEPTED_ISSUES_KEY, "High Impact Accepted Issues", Metric.ValueType.INT)
-    .setDescription("Accepted issues with high impact")
+  public static final Metric<Integer> HIGH_IMPACT_ACCEPTED_ISSUES =
```

Comment:
Is this metric going to be used again or is it still deprecated?

Reply:
It will be used again. We have a separate [MMF](https://sonarsource.atlassian.net/browse/MMF-4060) to remove all the deprecations related to old types (code smell, vulnerability, bug), severities, and custom rule/issue severity